src/odemis/gui/comp/overlay/world.py

```diff
@@ -268,17 +268,18 @@ def __init__(self, cnvs, roa=None, scanner=None, colour=gui.SELECTION_COLOUR):
           updated, and when its value changes, the rectangle is redrawn
           accordingly). Value is relative to the scanner.
         scanner (None or HwComponent): The scanner component to which the relative
-          ROA values refer to. Required if a roa argument is passed.
+          ROA values refer to. Required if a roa argument is passed and if the ROI is a ratio it should not be passed.
```

Comment:
Now, it not "required", as it's fine if it's None. So the following would be clearer: "If None, the roa argument is interpreted as absolute physical coordinates (m). If it's a HwComponent, the roa will be interpreted as a ratio of its fielf of view".
Also, update the docstring of `roa` too, to indicate "Value is relative to the scanner (if passed), and otherwise it's absolute (in m)".